supreme_court_predictions/data/statistics/DataCleaner.py

```diff
@@ -0,0 +1,225 @@
+        print(f"Data will be saved to {self.OUTPUT_PATH}")
+
+        if not downloaded_corpus:
+            print("Corpus not present, downloading...")
+            self.get_data()
+        else:
+            print("Corpus already downloaded")
+
+    def get_data(self):
+        """
+        Loads and outputs the Supreme Court Corpus data.
+        """
+
+        print("Loading Supreme Court Corpus Data...")
+        corpus = Corpus(filename=download("supreme-corpus"))
+        corpus.dump("supreme_corpus", base_path=self.LOCAL_PATH)
+
+    # Begin reading data
+    def load_data(self, file_name):
+        """
+        Opens the data and returns it as a dictionary.
+
+        :param file_name: The name of the file to open
+        :return: The data as a dictionary
+        """
+
+        path = self.LOCAL_PATH + f"/supreme_corpus/{file_name}"
+        if "jsonl" in file_name:
+            data = []
+            with open(path) as json_file:
+                json_list = list(json_file)
+
+            for json_str in json_list:
+                clean_json = json.loads(json_str)
+                data.append(clean_json)
+        else:
+            with open(path) as file:
+                data = json.load(file)
+        return data
+
+    def speakers_to_df(self, speakers_dict):
+        """
+        Converts the speakers dictionary to a pandas dataframe.
+
+        :param speakers_dict: The speaker's dictionary
+        :return: The speakers dataframe
+        """
+
+        dict_list = []
+        for speaker_key in list(speakers_dict.keys()):
+            speaker_data = speakers_dict[speaker_key]["meta"]
+            speaker_data["speaker_key"] = speaker_key
+            dict_list.append(speaker_data)
+
+        df = pd.DataFrame(dict_list)
+        df.rename(
+            columns={
+                "name": "speaker_name",
+                "type": "speaker_type",
+                "role": "speaker_role",
+            },
+            inplace=True,
+        )
+        return df
+
+    def get_conversation_dfs(self, conversations_dict):
+        """
+        Converts the conversations dictionary to several pandas dataframes.
+
+        :param conversations_dict: The conversations' dictionary
+        :return: The conversations dataframe, advocates dataframe,
+                and voters dataframe
+        """
+        metadata_list = []
+        advocates_list = []
+        voters_list = []
+
+        for conversation_id in list(conversations_dict.keys()):
+            clean_dict = {}
+            conversation_data = conversations_dict[conversation_id]["meta"]
+            clean_dict["id"] = conversation_id
+            clean_dict["case_id"] = conversation_data["case_id"]
+            clean_dict["winning_side"] = conversation_data["win_side"]
+
+            advocates = conversation_data["advocates"]
+            voters = conversation_data["votes_side"]
+
+            for advocate in advocates:
+                advocate_dict = {}
```

Comment:
Since you are building this all out below the instantiation, you should just do it all in one step:
```python
advocate_dict = {"id": conversation_id,
                                 "case_id": conversation_data["case_id"],
                                 "advocate": advocate,
                                 "side": advocates[advocate]["side"],
                                 "role": advocates[advocate]["role"]}
```

There are a few spots within this method and the next that you can do this, it would simplify the code a bit.